org.gnome.Maps.json

```diff
@@ -144,15 +144,13 @@
             "config-opts": [
                 "-Ddemos=false",
                 "-Dgtk_doc=false",
-                "-Dvapi=false",
-                "-Dlibsoup3=true",
-                "-Dvector_renderer=true"
```

Comment:
This does not match https://gitlab.gnome.org/GNOME/gnome-maps/-/blob/main/org.gnome.Maps.json?ref_type=heads, any reason for it?

Reply:
Right, yeah, vector_renderer=true is the default from libshumate 1.2 and on.

So, I thought it might be cleaner to remove the explicit setting here (thought we had already done that upstream, but probably forgot).

So, I think we should make this change there as well.

Reply:
I made the corresponding change in https://gitlab.gnome.org/GNOME/gnome-maps/-/commit/8907f26322e1c02ead9ec9f0669331a71592c1a7